### ISSUE TITLE
Odd behavior with OpenAPI v3 specs and form data adding an extra 'body' param

### ISSUE THREAD

**Issue by kelnos** [twilio/guardrail#237]
The [OpenAPI Form Data examples](https://swagger.io/docs/specification/describing-request-body/) include `type: object` when defining schemas for the form parameters.  When doing this, however, Guardrail emits the form params, plus an extra body param.  Unclear if this is intended (and if leaving out `type` is correct), or if this is a problem with the OpenAPI parser or with how Guardrail is interpreting the parser's output.  For reference, this triggers the issue:

```
openapi: 3.0.2
info:
  title: Whatever
  version: 1.0.0
paths:
  /foo:
    post:
      operationId: doFoo
      requestBody:
        required: true
        content:
          multipart/form-data:
            schema:
             type: object
              properties:
                bar:
                  type: integer
                  format: int64
                baz:
                  type: string
      responses:
        '200':
          description: OK
```

**Comment by dpblh** [comment]
I hope this helps you.
[https://github.com/twilio/guardrail/blob/67b94a0f97b6ff7a30d125c393bdb05ad916cc9a/modules/sample/src/main/resources/multipart-form-data.yaml](https://github.com/twilio/guardrail/blob/67b94a0f97b6ff7a30d125c393bdb05ad916cc9a/modules/sample/src/main/resources/multipart-form-data.yaml)

**Comment by kelnos** [comment]
@dpblh can you clarify?  I wrote that yaml file; not sure how it's relevant to this?

**Comment by dpblh** [comment]
@kelnos 
> Including `type` causes us to emit the three form parameters as expected, but also a fourth body parameter of an uspecified JSON object type.

From the file commentary it follows that by excluding `type` from yaml file, you can achieve the expected effect.
It works for me)

**Comment by kelnos** [comment]
@dpblh yes, I know.  That's the point of this issue; it _shouldn't_ do that.  `type: object` should be fine there.
